front_end/index.html

```diff
@@ -1,91 +1,46 @@
+		
+		<!-- This adds jquery (confirmed by: lucy + mira + mikalia) -->
+		<script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n" crossorigin="anonymous"></script>
+		
+		<!-- This adds more bootstrap stuff (confirmed by: kirby + mira)-->
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>
+		
+		<!-- This adds bootstrap functionality (confirmed by: kirby + mira) -->
+		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn" crossorigin="anonymous"></script>
+		
+		<!-- This adds momentjs resources (by: Kendell)-->
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.20.1/moment.min.js"></script>
+		
+		<!-- I linked your style to this html file this way (confirmed by: lucy)-->
 		<link rel="stylesheet" href="style.css" type="text/css" />
-	
-	<script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
 
 	</head>
-	
-	<!-- I moved your html inside the body of the page -->
 	<body>
+	
+		
+		<!-- This adds a digitial clock element (confirmed by:  lucy + mikilia) -->
+		<!-- <div id="clock" class="clock">Loading...</div> -->
+		
+		<!-- Analog clock here by:Kendell -->
 		<div class="hero-circle">
 			<div class="hero-face">
 				<div id="hour" class="hero-hour"></div>
 				<div id="minute" class="hero-minute"></div>
 				<div id="second" class="hero-second"></div>
 			</div>
 		</div>
-	<button type="check-in">Check In</button>
-	<button type="check-out">Check out</button>
+		
+		<!-- This adds check-in and check-out buttons (confirmed by: Johnny) -->
+  	<button id="check-in">Check In</button>
+  	<button id="check-out">Check out</button>
+  	
 	</body>
 	
-	<!-- We need to add links to the other resources(moment.js) and jquery here! -->
-
-	<!--
-		<script type="text/javascript" src="node_modlules/RESOURCE_NAME_HERE"></script>
-	-->
-	
-	<!-- I linked the javascript file to this html file, so it should run! -->
-	<script type="text/javascript" src="script.js"></script>
-	
-</html>
-
-<div>
-<img src="dg8.gif" name="hr1"><img 
-src="dg8.gif" name="hr2"><img 
-src="dgc.gif"><img 
-src="dg8.gif" name="mn1"><img 
-src="dg8.gif" name="mn2"><img 
-src="dgc.gif"><img 
-src="dg8.gif" name="se1"><img 
-src="dg8.gif" name="se2"><img 
-src="dgam.gif" name="ampm"></div>
-
-<script type="text/javascript">
-// (c) 2000-2014 ricocheting.com
-dg = new Array();
-dg[0]=new Image();dg[0].src="dg0.gif";
-dg[1]=new Image();dg[1].src="dg1.gif";
-dg[2]=new Image();dg[2].src="dg2.gif";
-dg[3]=new Image();dg[3].src="dg3.gif";
-dg[4]=new Image();[4].src="dg4.gif";
-dg[5]=new Image();[5].src="dg5.gif";
-dg[6]=new Image();[6].src="dg6.gif";
-dg[7]=new Image();[7].src="dg7.gif";
```

Comment:
why is this all red??
Why are there so many conflicts?

Reply:
these arent conflicts, they are deletions